### ISSUE TITLE
How do I use multiple azure containers?

### ISSUE THREAD

**Issue by epicfaace** [jschneier/django-storages#426]
For example, I'd like to use a different container for my DEFAULT_FILE_STORAGE and one for STATICFILES_STORAGE.

**Comment by Bastien-Brd** [comment]
+1

**Comment by Bastien-Brd** [comment]
Making (at least) the `AZURE_CONTAINER` an argument of the AzureStorage class constructor would allow several instances of the AxureStorage backend to be instantiated for different usages. 

For example, I use:
```
AZURE_STORAGE_CLASS = get_storage_class('storages.backends.azure_storage.AzureStorage')
file_storage_backend = AZURE_STORAGE_CLASS()

class MyModel(models.Model):
    file = models.FileField(storage=file_storage_backend)
```

but that forces that backend to use the one set of django settings variable. But making these variables arguments of the `AzureStorage` class at https://github.com/jschneier/django-storages/blob/master/storages/backends/azure_storage.py#L41 (defaulting to the django settings variable if not specified) would allow me to define several storage backends with different containers (or even different Azure accounts):

```
AZURE_STORAGE_CLASS = get_storage_class('storages.backends.azure_storage.AzureStorage')

static_files_storage_backend = AZURE_STORAGE_CLASS(azure_container='static_files_container')
user_uploaded_files_storage_backend = AZURE_STORAGE_CLASS(azure_container='user_uploaded_files_container')

class MyModel(models.Model):
    file = models.FileField(storage=static_files_storage_backend)

class MyOtherModel(models.Model):
    file = models.FileField(storage=user_uploaded_files_storage_backend)
```

**Comment by sww314** [comment]
Fixed in #565. New documentation includes an example of handling multiple containers.
